src/api/app/views/webui/subscriptions/_subscriptions_form.html.haml

```diff
@@ -1,17 +1,21 @@
 - i = 0
-- subscriptions_form.subscriptions_by_event.each do |event_class, subscriptions|
+- subscriptions_form.subscriptions_by_event.each do |subscription_by_event|
```

Comment:
```suggestion
- subscriptions_form.subscriptions_by_event.each_with_index do |subscription|
```

Reply:
Then you can also skip the `i`...

Reply:
In this case, we can't use `each_with_index` because the index will be reset in each event, and we don't want that.

Reply:
nitpick: It would still help readability if the variable would be called `subscription`

Reply:
Changed